diff --git a/CMake/folly-deps.cmake b/CMake/folly-deps.cmake
index 6077792..2198e62 100644
--- a/CMake/folly-deps.cmake
+++ b/CMake/folly-deps.cmake
@@ -69,7 +69,7 @@ endif()
 find_package(BZip2 MODULE)
 set(FOLLY_HAVE_LIBBZ2 ${BZIP2_FOUND})
 if (BZIP2_FOUND)
-  list(APPEND FOLLY_INCLUDE_DIRECTORIES ${BZIP2_INCLUDE_DIRS})
+  list(APPEND FOLLY_INCLUDE_DIRECTORIES ${BZIP2_INCLUDE_DIR})
   list(APPEND FOLLY_LINK_LIBRARIES ${BZIP2_LIBRARIES})
 endif()
 
